### DISCUSSION TITLE
Error fresh out of box

### DISCUSSION THREAD

**Discussion by sgtjueggmasterrr** [bassamanator/Sovol-SV06-firmware#61] [General]
Hello!

I loved your tutorial, but I am getting this error when trying to call START_PRINT

**_Error evaluating 'gcode_macro PRINT_START:gcode': jinja2.exceptions.UndefinedError: 'dict object' has no attribute 'BED'_**

[klippy(4).log](https://github.com/bassamanator/Sovol-SV06-firmware/files/11659666/klippy.4.log)

[moonraker.log](https://github.com/bassamanator/Sovol-SV06-firmware/files/11659665/moonraker.log)

![2023-06-05 23_28_54-Settings](https://github.com/bassamanator/Sovol-SV06-firmware/assets/135683809/f86820fa-9a46-46c0-80bc-dd52a23875ee)

**Comment by bassamanator** [comment]
You missed something at the end, check docs again.

**Reply by bassamanator** [reply]
Adjust your slicer section likely.

**Comment by sgtjueggmasterrr** [comment]
I don't think it is the slicer but I could be wrong. Here is the gcode

[heavy_metal_groot_01_0.2mm_PLA_SV06PLUS_49m.txt](https://github.com/bassamanator/Sovol-SV06-firmware/files/11659701/heavy_metal_groot_01_0.2mm_PLA_SV06PLUS_49m.txt)
![2023-06-05 23_46_44-_Untitled - PrusaSlicer-2 6 0-alpha6 based on Slic3r](https://github.com/bassamanator/Sovol-SV06-firmware/assets/135683809/9cb3592b-8ece-4b5b-a0fd-93164abd3c9d)

**Reply by sgtjueggmasterrr** [reply]
Well I did mess up the slicer settings... My bad!

For PrusaSlicer: https://ellis3dp.com/Print-Tuning-Guide/articles/passing_slicer_variables.html#prusa-slicer

Make sure if copying and pasting to correct any errors in formatting!

**Reply by bassamanator** [reply]
No worries. Please close the discussion when ready and star the repo if you found it useful. Best!
